pallets/committee/src/lib.rs

```diff
@@ -199,6 +195,8 @@ pub mod pallet {
 		ProposalNotAcceptedConstituentVeto,
 		/// Tried to close a proposal but proposal was denied by council
 		ProposalNotAcceptedCouncilDeny,
+		/// Attempted to execute a proposal that has already been closed
+		ProposalAlreadyClosed,
 		/// Attempted to execute a proposal that has already been executed
 		ProposalAlreadyExecuted,
```

Comment:
What's the difference between closed and executed?
This is rather confusing since the extrinsic is also named `closed`

Reply:
https://github.com/ChainSafe/PINT/pull/410#discussion_r714061234

Reply:
This terminology is a bit unfortunate because the `fn close` tries to _execute_ the proposal. 
how about TimedOut?